core/iwasm/libraries/lib-wasi-threads/tid_allocator.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+#ifndef _TID_ALLOCATOR_H
+#define _TID_ALLOCATOR_H
+
+#include "platform_common.h"
+
+#define TID_ALLOCATOR_INIT_SIZE CLUSTER_MAX_THREAD_NUM
```

Comment:
This is only used in `tid_allocator.c`, so I'd suggest defining it there; if there's a need to use it in other places in the future, we can move it back here.

Reply:
As for `TID_MIN` and `TID_MAX`, I put it in the header since they are some parameters we can change to alter the behavior of the id allocator without looking into the `.c` implementation